pkg/cmd/list.go

```diff
@@ -179,15 +180,22 @@ func NewWhoCanCommand(streams clioptions.IOStreams) (*cobra.Command, error) {
 			}
 
 			// Output check results
-			printer.PrintChecks(action, roleBindings, clusterRoleBindings)
+			output = strings.ToLower(output)
+			if output == outputJson {
+				printer.ExportData(action, roleBindings, clusterRoleBindings)
+			} else if output == outputWide || output == "" {
+				printer.PrintChecks(action, roleBindings, clusterRoleBindings)
+			} else {
+				return fmt.Errorf("Invalid output format: %v", output)
```

Comment:
NIT: Error string should not be capitalized or end with punctation
 
```suggestion
				return fmt.Errorf("invalid output format: %v", output)
```